app/src/main/kotlin/no/nav/k9punsj/utils/ServerRequestUtils.kt

```diff
@@ -22,6 +24,10 @@ object ServerRequestUtils {
     internal suspend fun ServerRequest.mapMatchFagsak() = body(BodyExtractors.toMono(Matchfagsak::class.java)).awaitFirst()
     internal suspend fun ServerRequest.mapMatchFagsakMedPerioder() = body(BodyExtractors.toMono(MatchFagsakMedPeriode::class.java)).awaitFirst()
 
+    internal suspend fun ServerRequest.mapSøknadTypeDto(søknadType: String) = when (søknadType) {
```

Comment:
Hvorfor er den flyttet hit? 
Den er ikke felles for hele prosjektet, men kun `korrigeringinntektsmelding`.
Den burde ligge der den brukes funksjonelt :)

Reply:
Flyttet tilbake efter slack-konvo.